app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -4,7 +4,7 @@
     {{input type='text' id='name' value=data.event.name}}
   </div>
   <div class="field">
-    <label for="location">{{t 'Location'}}</label>
+    <label class="required" for="location">{{t 'Location'}}</label>
```

Comment:
The issue is to show a message if the location, date or ticketing is missing and the user tries to publish the event. Introducing the "required" label is just misleading. The event can be stored as a draft if these informations are missing. You've to add the correct messages not make them required.

Reply:
@ritikamotwani I'm currently working on it. I'll remove `[WIP]` label as soon as it gets completed :sweat_smile: 
I've added `required` just for frontend check, nothing more. I'll also open a PR which will check these attributes on server before publishing the event.